File: libSOR/src/activemq/Gestor_activemq.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package activemq;

import activemq.Consumer;
import activemq.Producer;
import javax.jms.JMSException;
import org.apache.log4j.BasicConfigurator;

/**
 *
 * @author Cute
 */
public class Gestor_activemq {
     
       public     Producer producer;
       public     Consumer consumer;
        public    String nombreCola;
    public Gestor_activemq() throws JMSException {
         BasicConfigurator.configure();

    }
    public void create_Producer(String _nombreCola) throws JMSException{
    producer = new Producer(_nombreCola);
    }
    public void create_Consumer(String _nombreCola) throws JMSException{
    consumer = new Consumer(_nombreCola);
    }
    
    
}

File: desguaceNET/desguaceNET/libSOR/activemq/Gestor_activemq.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Apache.NMS.ActiveMQ;

namespace desguaceNET.libSOR.activemq
{
    class Gestor_activemq
    {
        public Consumer consumer;
        public string nombreCola;
        public Gestor_activemq()
        {
            //BasicConfigurator.configure();

        }

        public void create_Consumer(String _nombreCola)
        {
            consumer = new Consumer(_nombreCola);
        }
    }
}
